test/thermo/ThermoPhase_Test.cpp

```diff
@@ -109,4 +109,226 @@ TEST_F(TestThermoMethods, setState_AnyMap)
     EXPECT_NEAR(thermo->temperature(), 298.15, 1e-6);
 }
 
+TEST(TestMixtureMethods, getSet_EquilRatio_MixtureFraction)
```

Comment:
It's probably worth breaking this up into a few separate test cases -- basically, one for each of the places where you have a comment. You can put any parts that get reused between tests in a fixture class and then use the `TEST_F(TestClassName, testName)` style that you see in some other tests.